SECURITY.md

```diff
@@ -0,0 +1,9 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+Per the Linux Foundation Vulnerability Disclosure Policy, if you find a vulnerability in a project maintained by the Open Source Security Foundation (OpenSSF), please report that directly to the project maintaining that code, preferably using GitHub's Private Vulnerability Reporting.
```

Comment:
```suggestion
Per the Linux Foundation Vulnerability Disclosure Policy, if you find a vulnerability in a project maintained by the Open Source Security Foundation (OpenSSF), please [report that directly to the project](https://github.com/guacsec/sw-id-core/security).
```

Note that we don't actually have reporting enabled for this repo, so we might want to use `https://github.com/guacsec/guac/security/advisories/new` as the link target instead.

Reply:
Updated with a note to use the main repo.